src/app/presentation/presentation/presentation.component.ts

```diff
@@ -1,14 +1,15 @@
-import {Component, EventEmitter, Input, Output} from '@angular/core';
+import {Component, EventEmitter, Input, OnInit, Output} from '@angular/core';
 import {ActivatedRoute} from '@angular/router';
 import {Mode} from '../mode.enum';
 import {AnalyticsService} from '../analytics.service';
+declare const ga;
```

Comment:
As much as I'm not a fan of this name, this is a convention to call google analytics this way, e.g. https://developers.google.com/analytics/devguides/collection/analyticsjs/events